index.js

```diff
@@ -56,6 +79,38 @@ function assembleStyles () {
 		});
 	});
 
+	styles.color = {
+		ansi16: {},
+		ansi256: {},
+		ansi16m: {}
+	};
+
+	styles.bgColor = {
+		ansi16: {},
+		ansi256: {},
+		ansi16m: {}
+	};
+
+	for (var key in colorConvert) {
+		var fn = colorConvert[key];
+		var conversion = key.replace(/^([^2]+).+$/, '$1');
+		if (/2ansi16$/.test(key)) {
+			styles.color.ansi16[conversion] = wrapAnsi16(fn, 0);
+			styles.bgColor.ansi16[conversion] = wrapAnsi16(fn, 10);
+			continue;
+		}
+		if (/2ansi$/.test(key)) {
+			styles.color.ansi256[conversion] = wrapAnsi256(fn, 0);
+			styles.bgColor.ansi256[conversion] = wrapAnsi256(fn, 10);
+			continue;
+		}
+		if (/2rgb$/.test(key)) {
+			styles.color.ansi16m[conversion] = wrapAnsi16m(fn, 0);
+			styles.bgColor.ansi16m[conversion] = wrapAnsi16m(fn, 10);
+			continue;
+		}
```

Comment:
It's possible but the tradeoff would be more code at the top. It'd probably even amount to more LOC in the end anyway. This is probably fine for now.